quad_control/scripts/TrajectoryPlanner/fixed_point_trajectory.py

```diff
@@ -18,6 +19,15 @@ def description(cls):
     def get_parameters_names(cls):
         return ()
         
+    @classmethod
+    def parameters_to_string(cls):
+        dic = dict()
+        return json.dumps(dic)
+        
+    @classmethod
+    def string_to_parameters(cls, string):
+        return dict()
+        
```

Comment:
This should fix the error when trying to construct a `FixedPointTrajectory` from the GUI